src/openmp/Level.cpp

```diff
@@ -0,0 +1,45 @@
+#include "openmp/Level.h"
+
+namespace wasm {
```

Comment:
As `openmp` is now a top-level `src` directory I think the namespace should just be `openmp` rather than `wasm::openmp`

Reply:
it doesn't need to be top level anymore, I am moving this down to a subdirectory in was. I originally forced this to be a nested namespace to remind me to change it later, but since you seem to be tying namespaces to directory hierarchy I'll follow that and move it

Reply:
:+1: